index.html

```diff
@@ -0,0 +1,359 @@
+            alt="icon"
+          />
+          <p
+            class="discover-more__content-title"
+          >
+            Sales Tracking
+          </p>
+          <p class="discover-more__content-description">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+        <div class="discover-more__content-info">
+          <img
+            class="discover-more__content-icon"
+            src="img/Feature Icon1.png"
+            alt="icon"
+          />
+          <p
+            class="discover-more__content-title discover-more__content-title--right"
+          >
+            Sales Tracking
+          </p>
+          <p class="discover-more__content-description">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+        <div class="discover-more__content-info">
+          <img
+            class="discover-more__content-icon"
+            src="img/Feature Icon1.png"
+            alt="icon"
+          />
+          <p
+            class="discover-more__content-title discover-more__content-title--right"
+          >
+            Sales Tracking
+          </p>
+          <p class="discover-more__content-description">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="features">
+      <h2 class="features__title">FEATURES</h2>
+
+      <div class="features__cards">
+        <div class="features__card">
+          <img class="features__card-icon" src="img/Icon@1x.png" alt="icon" />
+
+          <p class="features__card-title">
+            Advanced 256-bit encryption
+          </p>
+          <p class="features__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+        <div class="features__card">
+          <img class="features__card-icon" src="img/Icon@1x.png" alt="icon" />
+
+          <p class="features__card-title">
+            Advanced 256-bit encryption
+          </p>
+          <p class="features__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+        <div class="features__card">
+          <img class="features__card-icon" src="img/Icon@1x.png" alt="icon" />
+
+          <p class="features__card-title">
+            Advanced 256-bit encryption
+          </p>
+          <p class="features__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="testimonials">
+      <p class="testimonials__title ">
+        The stunning results our customers have experienced
+      </p>
+      <div class="testimonial__cards">
```

Comment:
You don't have a block called `testimonial` but it is used here! but you have one called `testimonials`